cmd/thanos/rule.go

```diff
@@ -75,6 +78,9 @@ func registerRule(m map[string]setupFunc, app *kingpin.Application, name string)
 
 	objStoreConfig := regCommonObjStoreFlags(cmd, "")
 
+	filesToWatch := cmd.Flag("store.file-sd-config", "Path to file that contain addresses of query peers. The path can be a glob pattern (repeatable).").
```

Comment:
`store.file-sd-config` > `store.file-sd-config.files`? or is that just over-complicating things?